### PR TITLE
Doc: Updating the landing page, move the collection below AVD umbrella

### PR DESCRIPTION
**Pull request by JulioPDX**
## Change Summary

Updating landing page to move the Ansible collection below the AVD umbrella.

## Proposed changes

- Update layout to clarify The Ansible collection vs AVD as a framework

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by JulioPDX** [comment]
@carlbuchmann Work in progress to move the ansible collection below "AVD." Let me know what you think, I will also update the GitHub readme after changes are approved.

File: mkdocs.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Project information
-site_name: Arista AVD collection
+site_name: Arista AVD
 site_author: Arista Ansible Team
 site_description: Arista Validated Design collection's documentation
```
Comment (ClausHolbechArista) [inline mkdocs.yml]: Still says collection

File: ansible_collections/arista/avd/README.md [hunk 2]
```diff
@@ -19,7 +14,16 @@ Full documentation for the collection:
...
 
-## Roles overview
+## AVD Ansible Collection
+
+[Arista Networks](https://www.arista.com/) supports Ansible for managing devices running Arista's **Extensible Operating System (EOS)** natively through it's **EOS API (eAPI)** or [**CloudVision Portal (CVP)**](https://www.arista.com/en/products/eos/eos-cloudvision). The collection includes a set of Ansible roles and modules to help kick-start your automation with Arista. The various roles and templates provided are designed to be customized and extended to your needs.
+
+Full documentation for the collection:
+
+- [stable version](https://www.avd.sh/en/stable/)
+- [collection development version](https://www.avd.sh/en/devel/)
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]:
```suggestion
- [development version](https://www.avd.sh/en/devel/)
```

File: ansible_collections/arista/avd/README.md [hunk 3]
```diff
@@ -59,7 +63,7 @@ Ansible galaxy hosts all stable versions of this collection. Installation from a
 
 ## Ask a question
 
-Support for this `arista.avd` collection is provided by the community directly in this repository. If you have any questions, please leverage the GitHub [discussions board](https://github.com/aristanetworks/ansible-avd/discussions).
+Support for the `arista.avd` collection is provided by the community directly in this repository. If you have any questions, please leverage the GitHub [discussions board](https://github.com/aristanetworks/ansible-avd/discussions).
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md:66]: Should we mention that if you have a support agreement you can contact Arista support?

File: ansible_collections/arista/avd/README.md [hunk 4]
```diff
@@ -1,13 +1,8 @@
...
 
 <center><img src="media/avd-logo.png" alt="Arista AVD Overview" width="800"/></center>
 
-[Arista Networks](https://www.arista.com/) supports Ansible for managing devices running Arista's **Extensible Operating System (EOS)** natively through it's **EOS API (eAPI)** or [**CloudVision Portal (CVP)**](https://www.arista.com/en/products/eos/eos-cloudvision). This collection includes a set of Ansible roles and modules to help kick-start your automation with Arista. The various roles and templates provided are designed to be customized and extended to your needs.
-
-Full documentation for the collection:
-
-- [stable version](https://www.avd.sh/en/stable/)
-- [collection development version](https://www.avd.sh/en/devel/)
+Arista Validated Designs (AVD) is an extensible data model that defines Arista’s Unified Cloud Network architecture as “code.”
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]:
```suggestion
Arista Validated Designs (AVD) is an extensible data model that defines Arista’s Unified Cloud Network architecture as "code".
```

File: ansible_collections/arista/avd/README.md [hunk 5]
```diff
@@ -35,20 +39,20 @@ This repository provides content for Arista's **arista.avd** collection. The fol
...
 
-- [Arista AVD Plugins](plugins/README.md)
+### Examples
 
-## Collection installation
+- [Getting started](./docs/getting-started/intro-to-ansible-and-avd.md)
+- [Arista NetDevOps GitHub repository](https://github.com/aristanetworks/netdevops-examples)
 
-Ansible galaxy hosts all stable versions of this collection. Installation from ansible-galaxy is the most convenient approach for consuming `arista.avd` content. Please follow the collection installation [guide](https://avd.sh/en/stable/docs/installation/collection-installation.html).
+## Custom plugins & modules
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]: This is still part of the collection
```suggestion
### Custom plugins & modules
```

File: ansible_collections/arista/avd/README.md [hunk 6]
```diff
@@ -1,13 +1,8 @@
...
 
 <center><img src="media/avd-logo.png" alt="Arista AVD Overview" width="800"/></center>
 
-[Arista Networks](https://www.arista.com/) supports Ansible for managing devices running Arista's **Extensible Operating System (EOS)** natively through it's **EOS API (eAPI)** or [**CloudVision Portal (CVP)**](https://www.arista.com/en/products/eos/eos-cloudvision). This collection includes a set of Ansible roles and modules to help kick-start your automation with Arista. The various roles and templates provided are designed to be customized and extended to your needs.
-
-Full documentation for the collection:
-
-- [stable version](https://www.avd.sh/en/stable/)
-- [collection development version](https://www.avd.sh/en/devel/)
+Arista Validated Designs (AVD) is an extensible data model that defines Arista’s Unified Cloud Network architecture as "code".
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/README.md]:
```suggestion
Arista Validated Designs (AVD) is an extensible data model that defines Arista's Unified Cloud Network architecture as "code".
```

**Comment by ClausHolbechArista** [comment]
@JulioPDX

File: ansible_collections/arista/avd/README.md (hunk 3 above)
Comment (JulioPDX) [reply ansible_collections/arista/avd/README.md:66]: @ClausHolbechArista Just pushed a small note on this topic.  Please let me know if it requires any edits

File: mkdocs.yml (hunk 1 above)
Comment (ClausHolbechArista) [inline mkdocs.yml]:
```suggestion
site_description: Arista Validated Designs documentation
```

Comment (ClausHolbechArista) [reply mkdocs.yml]: You resolved this one with no update. I have comitted a change to line 4 to remove the "collection's" and add "s" on "Designs". I will approve now, but let me know if there was a reason why you didn't change the description.

**Review by ClausHolbechArista — APPROVED** [review]

**Review by nielsjlarsen — APPROVED** [review]
LGTM!

**Pull request merged**